scripts/qa_crowbarsetup.sh

```diff
@@ -491,10 +492,17 @@ function onadmin_prepare_sles_repos()
         add_mount "SUSE-Cloud-SLE-11-SP3-deps/sle-11-x86_64/" "" "${targetdir_install}" "Cloud-Deps"
         zypper_refresh
     else
-        zypper se -s sles-release|grep -v -e "sp.up\s*$" -e "(System Packages)" |grep -q x86_64 || zypper ar http://$susedownload/install/SLP/SLES-${slesversion}-LATEST/x86_64/DVD1/ sles
+        zypper se -s sles-release | \
+            grep -v -e "sp.up\s*$" -e "(System Packages)" | \
+            grep -q x86_64 \
+        || zypper ar \
+            http://$susedownload/install/SLP/SLES-${slesversion}-LATEST/x86_64/DVD1/ \
+            sles
 
         if ! $longdistance ; then
-            add_mount "" "clouddata.cloud.suse.de:/srv/nfs/suse-$suseversion/install" "${targetdir_install}"
+            add_mount "" \
```

Comment:
Conventional wisdom across the industry is that it is a bad idea to exceed 80 column lines.  This is visible in countless areas, from the Linux kernel coding style through to PEP8 and beyond.  Long lines definitely make code harder to read for me, so even if you don't feel it increases readability for you then please be considerate of other people.

Also don't forget that some user interfaces do not automatically wrap lines.  Which code in the below image do you think is more readable - the red or the green?

![line-chopped](https://cloud.githubusercontent.com/assets/100738/6230142/4063f500-b6b5-11e4-8e75-2e70c5f0af92.png)